test-frame-metrics-collector/src/main/java/io/skodjob/testframe/MetricsCollector.java

```diff
@@ -107,6 +110,16 @@ public Builder withScraperPodName(String scraperPodName) {
             return this;
         }
 
+        /**
+         * Deploy own scraper pod instead of using already created with name
+         *
+         * @return this builder instance to allow for method chaining
+         */
+        public Builder withOwnScraperPod() {
```

Comment:
```suggestion
        public Builder withDeployScraperPod() {
```
would be maybe better?